webapp-django/crashstats/supersearch/tests/test_views.py

```diff
@@ -851,6 +851,16 @@ def test_get_report_list_parameters(self):
 
         eq_(res['build_id'], ['12345'])
 
+        # Test missing parameters don't break.
```

Comment:
"break" is ambiguous. Can you replace with with `Test missing parameters don't raise an exception` because that's really what you're after, isn't it?